test/tsconfig.json

```diff
@@ -13,6 +13,7 @@
         "cli/watch",
         "transpile/directories",
         "transpile/outFile",
-        "../src/lualib"
+        "../src/lualib",
+        "../helpers"
```

Comment:
`helpers` is meant to be included only in scripts running in the Lua context, while our tests run in node. If we do need to include it in some `tsconfig.json` (which we would need for ESLint), `lualib` is probably the best one.